oraichain/assetlist.json

```diff
@@ -50,6 +50,526 @@
+          "exponent": 0
+        },
+        {
+          "denom": "scatom",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "cw20",
+      "address": "orai19q4qak2g3cj2xc2y3060t0quzn3gfhzx08rjlrdd3vqxhjtat0cq668phq",
+      "base": "cw20:orai19q4qak2g3cj2xc2y3060t0quzn3gfhzx08rjlrdd3vqxhjtat0cq668phq",
+      "name": "scATOM",
+      "display": "scatom",
+      "symbol": "scATOM",
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/scatom.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/scatom.svg"
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/scatom.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/scatom.svg"
+      }
+    },
+    {
+      "description": "The xOCH token is the cw20 token for the Oraichain.",
+      "denom_units": [
+        {
+          "denom": "cw20:orai1lplapmgqnelqn253stz6kmvm3ulgdaytn89a8mz9y85xq8wd684s6xl3lt",
+          "exponent": 0
+        },
+        {
+          "denom": "xoch",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "cw20",
+      "address": "orai1lplapmgqnelqn253stz6kmvm3ulgdaytn89a8mz9y85xq8wd684s6xl3lt",
+      "base": "cw20:orai1lplapmgqnelqn253stz6kmvm3ulgdaytn89a8mz9y85xq8wd684s6xl3lt",
+      "name": "Xoch",
+      "display": "xoch",
+      "symbol": "xOCH",
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/och.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/och.svg"
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/och.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/och.svg"
+      }
+    },
+    {
+      "description": "The wETH token is the cw20 token for the Oraichain.",
+      "denom_units": [
+        {
+          "denom": "cw20:orai1dqa52a7hxxuv8ghe7q5v0s36ra0cthea960q2cukznleqhk0wpnshfegez",
+          "exponent": 0
+        },
+        {
+          "denom": "weth",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "cw20",
+      "address": "orai1dqa52a7hxxuv8ghe7q5v0s36ra0cthea960q2cukznleqhk0wpnshfegez",
+      "base": "cw20:orai1dqa52a7hxxuv8ghe7q5v0s36ra0cthea960q2cukznleqhk0wpnshfegez",
+      "name": "Weth",
+      "display": "weth",
+      "symbol": "wETH",
+      "images": [
+        {
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/eth-white.svg",
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/eth-white.png"
+        },
+        {
```

Comment:
You have the same image object twice? There are different images for ETH, but you're referring to the same image twice